_examples/golang-basics/example.ridl

```diff
@@ -102,6 +102,10 @@ service ExampleService
   @internal
   - Status() => (status: bool)
   - Version() => (version: Version)
+  # Get user endpoint
+  #
+  # gives you basic info about user
+  @deprecated:GetUserV2
```

Comment:
Looks OK.. but I'm not sure if using the value in `@depracated:{value}` should have the meaning of the new endpoint name

Reply:
Maybe not :man_shrugging: 

I guess comments are enough ?

Reply:
I'm not sure. `@deprecated:GetUserV2` just doesn't feel right :)

Maybe there's a better name we will come up with "deprecated by" or "replaced by" or "use instead". Or let's just stick with comment for now until we get it right?